src/components/ApplicationsModal/styles.ts

```diff
@@ -526,6 +527,15 @@ export default makeStyles((theme) => ({
     width: "460px",
   },
 
+  markdownIcon: {
+    bottom: theme.spacing(1),
+    color: theme.palette.text.hint,
+    height: theme.spacing(3),
```

Comment:
Is the **height** and **width** suppose to be dynamic/change if the spacing changes?